src/components/ItemList.js

```diff
@@ -15,7 +24,7 @@ const ItemList = () => {
     const unsubscribe = onSnapshot(q, (querySnapshot) => {
       const items = [];
       querySnapshot.forEach((doc) => {
-        items.push(doc.data());
+        items.push({ data: doc.data(), id: doc.id });
```

Comment:
Do you want to update our firebasesnapshot hook to do this so you can use the hook here? Might help mitigate our inevitable merge conflict a bit since Cynthia and I already brought the hook in on this component in our branch :)

Reply:
We were definitely wondering about that and were going to ask about it in OH tonight!